### PR TITLE
README update

### PR DESCRIPTION
**Pull request by danhartley**
Adding contributor help for setting up the project, and for adding content and making a pull request

### CODE REVIEW

**Comment by danhartley** [comment]
[Update README](https://trello.com/c/6YU3EcOw/26-update-readme)

File: README.md [hunk 1]
```diff
@@ -8,6 +8,124 @@ The website was created in [Jekyll](https://jekyllrb.com/), a static website gen
...
+
+### Running the website locally
+
+  Clone the repo
+  ```sh
+  git clone https://github.com/ResponsibleTechWork/RespTechWork-website.git
+  ```
+  For Ruby 3.0.0 and above, add a dependency to webrick
+  ```sh
+  bundle add webrick
```
Comment (ialja) [inline README.md:34]: Now that `webrick` has been added to the Gemfile, the general instruction should be to run `bundle install`.

File: README.md [hunk 2]
```diff
@@ -8,6 +8,124 @@ The website was created in [Jekyll](https://jekyllrb.com/), a static website gen
 
 The design is based on the [Libris theme](https://github.com/stackbit-themes/libris-jekyll) by [Stackbit](https://www.stackbit.com/). The theme's SASS files are stored in the `_sass/imports` folder.
 
+## Getting started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+  Install or update ruby
```
Comment (ialja) [inline README.md:17]: Maybe mention that Jekyll requires Ruby version 2.5.0 or higher (easily checked using `ruby -v`).

File: README.md (hunk 1 above)
Comment (danhartley) [reply README.md:34]: Ay yes, tricky to test the setup before the setup is in place ;-) We don't even need bundle install…

Comment (ialja) [reply README.md:34]: I think we need it because we're using extra gems that are not included in Jekyll by default, so that's a good step to add.

File: README.md (hunk 2 above)
Comment (danhartley) [reply README.md:17]: Yes, that makes sense. If the user has Ruby 3 'bundle exec jekyll serve' will pull in the dependency on webrick.

File: README.md [hunk 3]
```diff
@@ -8,6 +8,39 @@ The website was created in [Jekyll](https://jekyllrb.com/), a static website gen
 
 The design is based on the [Libris theme](https://github.com/stackbit-themes/libris-jekyll) by [Stackbit](https://www.stackbit.com/). The theme's SASS files are stored in the `_sass/imports` folder.
 
+## Getting started on iOS
```
Comment (ialja) [inline README.md:11]: macOS :)

File: .gitignore [hunk 4]
```diff
@@ -3,3 +3,4 @@ _site
 .jekyll-cache
 .jekyll-metadata
 vendor
+CONTRIBUTING.md
```
Comment (ialja) [inline .gitignore:6]: Was this addition intentional?

Comment (danhartley) [reply .gitignore:6]: Yes. I want to have a working version with the bundle in, etc. before adding anything else; I didn't want to include this file. Probably should have stashed. I'll remove when I add the CONTRIBUTING.md file.

File: README.md (hunk 3 above)
Comment (danhartley) [reply README.md]: Good point! Yes, will change.
